File: prak3_php/Ram.php
<?php

class Ram{

    private $capacity;
    private $price;

    //konstruktor
    function __construct($capacity, $price){
        $this->capacity = $capacity;
        $this->price = $price;
    }

    //set
    public function setCapacity($capacity){
        $this->capacity = $capacity;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    //get
    public function getCapacity(){
        return $this->capacity;
    }

    public function getPrice(){
        return $this->price;
    }

    //destruktor
    function __destruct(){
    }

}


?>

File: prak3_python/Ram.py
from logging import captureWarnings


class Ram:

    def __init__(self, capacity, price):
    #konstruktor
        self.capacity = capacity
        self.price = price

    #set
    def setCapacity(self, capacity):
        self.capacity = capacity

    def setPrice(self, price):
        self.price = price

    #get
    def getCapacity(self):
        return self.capacity

    def getPrice(self):
        return self.price
